src/lib/download.js

```diff
@@ -22,6 +30,14 @@ export default (filename) => {
       fullUrl,
       (response) => {
         let cur = 0
+        const responseType = response.headers['content-type']
+        if (!responseType.includes('audio')) {
+          console.log((clk.red(`No audio file found: ${clk.magenta(fullUrl)} \ntrying again in ${retryInterval.asMinutes()} minutes.`)))
+          return setTimeout(() => (
+            download(filename, attempts - 1)
+            .catch(err => console.log(err))
```

Comment:
best to bubble up this error so it gets catched in the main promise flow (`downloader.js`)